CHANGELOG.md

```diff
@@ -6,20 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [14.0.2]
+## [14.0.2] - 2021-07-28
 ### Changed
 - Fix `resetPolling` functionality ([#546](https://github.com/MetaMask/controllers/pull/546))
-- TokenService improvements ([#541](https://github.com/MetaMask/controllers/pull/541))
```

Comment:
I don't think this is covered by previous change entries :thinking: "TokenService improvements" is pretty vague though.

Maybe something like "Improve token list API error handling" would be better.